private/protoplugin/runtime.go

```diff
@@ -34,7 +36,27 @@ type Runtime struct {
 	MethodIdempotency  *Symbol
 }
 
-func newRuntime(symbolPool *symbolPool, runtimeImportPath string, syntax ProtoSyntax) *Runtime {
+func newRuntime(symbolPool *symbolPool, runtimeImportPath string, syntax ProtoSyntax, bootstrapWKT bool) *Runtime {
+	if bootstrapWKT {
+		return &Runtime{
+			ProtoN:             symbolPool.new(syntax.String(), fmt.Sprintf("./%s.js", syntax.String())),
+			Message:            symbolPool.new("Message", "./message.js"),
+			PartialMessage:     symbolPool.new("PartialMessage", "./message.js").ToTypeOnly(),
+			PlainMessage:       symbolPool.new("PlainMessage", "./message.js").ToTypeOnly(),
+			FieldList:          symbolPool.new("FieldList", "./field-list.js").ToTypeOnly(),
+			MessageType:        symbolPool.new("MessageType", "./message-type.js").ToTypeOnly(),
+			BinaryReadOptions:  symbolPool.new("BinaryReadOptions", "./binary-format.js").ToTypeOnly(),
+			BinaryWriteOptions: symbolPool.new("BinaryWriteOptions", "./binary-format.js").ToTypeOnly(),
+			JsonReadOptions:    symbolPool.new("JsonReadOptions", "./json-format.js").ToTypeOnly(),
+			JsonWriteOptions:   symbolPool.new("JsonWriteOptions", "./json-format.js").ToTypeOnly(),
+			JsonValue:          symbolPool.new("JsonValue", "./json-format.js").ToTypeOnly(),
+			JsonObject:         symbolPool.new("JsonObject", "./json-format.js").ToTypeOnly(),
+			ProtoInt64:         symbolPool.new("protoInt64", "./proto-int64.js"),
+			ScalarType:         symbolPool.new("ScalarType", "./field.js"),
+			MethodKind:         symbolPool.new("MethodKind", "./service-type.js"),
+			MethodIdempotency:  symbolPool.new("MethodIdempotency", "./service-type.js"),
+		}
+	}
```

Comment:
Couldn't think of a better way to handle this without doing some significant rearchitecting. I think it's sufficient, if a little frail.